lib/ReactViews/FeatureInfo/feature-info-panel.scss

```diff
@@ -22,9 +22,10 @@
   a{
     @include link($feature-info-bg);
   }
-  @media screen and (min-width : $md){
+
+  @media screen and (min-width : $sm){
     //temp
-    max-width: calc(100vw - #{$work-bench-width} - 150px);
+    max-width: 432px;
```

Comment:
Is this change intentional?  It's not mentioned in the description of the PR.

Reply:
Sorry, updated PR description